### PR TITLE
Added Flames Game

### LINKED ISSUE #3822: [New game]: FLames game?
### 🎮 Game Request

Game logic and basic description
Describes relation between two people.

### Point down the features

Game points
INputs of girl and boys name , and output is their future relationship.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Saipradyumnagoud**
## PR Description 📜

Added Flames game where user can give there name and crush name to find the relation

Fixes #3822 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [ ] I have specified the respective issue number for which I have requested the new game.

<hr>

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Saipradyumnagoud ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Saipradyumnagoud,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Saipradyumnagoud,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Saipradyumnagoud,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Saipradyumnagoud,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Saipradyumnagoud,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Comment by Saipradyumnagoud** [comment]
What should i modify in readme

**Comment by Saipradyumnagoud** [comment]
I have added assets/images directory 
I have added Readme.md file also
could you please tell me what else should I do

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Saipradyumnagoud,
You need to make the following changes:
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

File: README.md [hunk 1]
```diff
@@ -278,8 +278,7 @@ This repository also provides one such platforms where contributers come over an
 | [Arkanoid_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Arkanoid_Game)               |
 | [Catch_Stars](https://github.com/Kunjgit/GameZone/tree/main/Games/Catch_Stars)               |
 | [LaserDarts] (https://github.com/Jagpreet153/GameZone/tree/main/Games/LaserDarts)
-| [Block Building](https://github.com/kunjgit/GameZone/tree/main/Games/Block_Building)                                 |
-|[Flames-Game](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Flames-game) |
```
Comment (Saipradyumnagoud) [inline README.md]: I have modified  Main ReadMe file

**Review by Saipradyumnagoud — COMMENTED** [review]
I Have modified main ReadMe File

**Comment by Saipradyumnagoud** [comment]
> Hello @Saipradyumnagoud, You need to make the following changes:
> 
> * Please modify the main README.md file.
> 
> Hoping that you will make those changes soon 🚀

I Have modified Main ReadMe.md file please check it

**Comment by kunjgit** [comment]
@Saipradyumnagoud you are having submodule in your code make sure you are following proper guidelines !
make PR again after that!

**Pull request closed without merging**